### DISCUSSION TITLE
TeXPage 提示“获取上传 Token 失败，请重试”

### DISCUSSION THREAD

**Discussion by Yxiaofa** [nju-lug/NJUThesis#250] [Q&A]
上传pdf图片时显示“获取上传 Token 失败，请重试!”

**Comment by atxy-blip** [comment]
TeXPage 的文件管理依赖腾讯云，现在是腾讯云挂了乱成一锅粥，等他们上游修好吧（

------

16:33 更新：已经修好了。
